R/link_flowlines.R

```diff
@@ -17,10 +18,27 @@
 #'
 #' @export
 
-link_to_flowlines = function(lats, lons, ids, max_dist = 100){
+link_to_flowlines = function(lats, lons, ids, max_dist = 100, dataset = "nhdh"){
+  dl_file = ""
+  id_column = ""
   bbdf_streams = NULL
-  load(file=system.file('extdata/nhd_bb_streams_cache.Rdata', package='hydrolinks'))
-  wbd_bb = bbdf_streams
+  if(tolower(dataset) == "nhdh"){
+    load(file=system.file('extdata/nhd_bb_streams_cache.Rdata', package='hydrolinks'))
+    dl_file = "extdata/nhdh.csv"
+    id_column = "PERMANENT_"
+    wbd_bb = bbdf_streams
+  }
+  else if(tolower(dataset) == "nhdplusv2"){
+    load(file=system.file('extdata/nhdplus_flowline_bb_cache.rdata', package='hydrolinks'))
+    dl_file = "extdata/nhdplusv2.csv"
+    id_column = "COMID"
+    wbd_bb = bbdf_flowline
+  }
+  else{
+    stop("Invalid dataset name!")
```

Comment:
Could you tell the user the proper options here? You can use `match.arg` earlier to make it quick and easy.